signature/spec/README.md

```diff
@@ -0,0 +1,118 @@
+# OCI Image Signature Specification
+
+**Version 0.1**
+
+## Introduction
+
+This document specifies an out-of-band cryptographic verification model for
+container images. Detached signatures are associated with images by manifest digest hash value.
+
+## Signature Format
+
+### Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="critical"></a>critical | [ [CriticalObject](#criticalObject) ] | **Required.**
+<a name="optional"></a>optional | [ [OptionalObject](#optionalObject) ] | **Optional.**
+
+#### <a name="criticalObject"></a>Critical Object
+
+Field Name | Type | Description
+---|:---:|---
+<a name="identity"></a>identity | [ [IdentityObject](#identityObject) ] | **Required.**
+<a name="image"></a>image | [ [ImageObject](#imageObject) ] | **Required.**
+<a name="type"></a>type | `string` | **Required.** Valid values: "atomic container signature"
+
+#### <a name="identityObject"></a>Identity Object
+
+Field Name | Type | Description
+---|:---:|---
+<a name="docker-reference"></a>docker-reference | `string` | **Required.** Image name
+
+#### <a name="imageObject"></a>Image Object
+
+Field Name | Type | Description
+---|:---:|---
+<a name="docker-manifest-digest"></a>docker-manifest-digest | `digest` | **Required.** Manifest digest in the form of `<algorithm>:<hashValue>`
```

Comment:
References to other CAS blobs should be by [descriptor][],

[descriptor]: https://github.com/opencontainers/image-spec/blob/v0.5.0/descriptor.md#oci-content-descriptors